s3/filestorage.go

```diff
@@ -290,7 +290,7 @@ func (s *SimpleStorageService) uploadMultipart(
 
 	// The following is loop is very similar to io.Copy except the
 	// destination is the s3 bucket.
-	for partNum = 1; partNum < maxPartNum; partNum++ {
+	for partNum++; partNum < maxPartNum; partNum++ {
```

Comment:
This line struct me as a little odd. Is it supposed to start from two?

Reply:
yes, the 1 index is already "used" above.